runtime.md

```diff
@@ -54,6 +49,10 @@ Note: The lifecycle is a WIP and it will evolve as we have more use cases and mo
 
 OCI compliant runtimes MUST support the following operations, unless the operation is not supported by the base operating system.
 
+All operations take a `<container-id>` as an argument, and all but [start](#start) use that container ID to [retrieve the container state](#query-state) needed to interact with the container.
+Those operations MUST successfully retrieve their required state when called under [conditions in which a state query MUST succeed](#query-state).
+Those operations MAY fail when called under conditions in which a state query MAY fail.
```

Comment:
While these operations will, clearly, need to interact with the state of the container to get their job done - I think its an implementation detail that doesn't need to be called out in the spec. As long as we clearly state what each operation needs to do then we're covered.